app/views/create_project.py

```diff
@@ -0,0 +1,74 @@
+from flask import session, redirect, url_for, request, render_template, flash
+from app import app, forms
+from app.util import is_logged_in, error_message
+import pymesync
+import re
+
+
+@app.route('/projects/create', methods=['GET', 'POST'])
+def create_project():
+    if not is_logged_in():
+        if request.method == 'GET':
+            return redirect(url_for('login', next=request.url_rule))
+        elif request.method == 'POST':
+            return "Not logged in.", 401
+
+    form = forms.CreateProjectForm()
+
+    ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
+                           test=app.config['TESTING'],
+                           token=session['token'])
+
+    users = ts.get_users()
+    error_message(users)
+
+    usernames = []
+    for user in users:
+        usernames.append((user['username'], user['display_name']))
+
+    form.members.choices = usernames
+    form.managers.choices = usernames
+    form.spectators.choices = usernames
```

Comment:
I don't understand how this works. Why would the members, managers, and spectators all be the same list of users?